packages/shared-components/src/pages/js/main.js

```diff
@@ -0,0 +1,18 @@
+let headerContext = document.createElement('onto-test-context');
+document.body.appendChild(headerContext);
+
+// Mock the navigateUrl function which is exposed by the single-spa via the root-config module
+// in order to allow the menu to work without going anywhere when clicking the menu items
+window.singleSpa = {
+  navigateToUrl: function (url) {
+    console.log('%cnavigate', 'background: blue', url);
```

Comment:
If this logger is just forgotten, then delete it

Reply:
deleted

Reply:
deleted